web/frontend/components/AnnotationCloner.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+<div id="annotation-cloner">
+  <button class="action annotate-casebook-nodes" data-disable-with="Clone-Node" v-on:click="displayModal()">Annotate {{properType()}}</button>
+  <Modal v-if="showModal"
+            @close="showModal = false">
+    <template class="modal-title" slot="title">
+      <span class="annotation-icon"></span>
+      <h4>Copying {{properType()}} for Annotation</h4>
+    </template>
+    <template class="modal-body-annotation" slot="body">
+      <p class="annotation-cloner-text"> <b> To Annotate, you need to Create a Copy.<br/><br/> Choose a book</b>  you would like  "<em>{{sectionSource}}</em>" to be copied into : </p>
+      <ul class="annotation-target-list">
+        <li v-for="cb in casebookTargets" v-bind:key="cb.form_target">
+          <form :action="cb.form_target" method="POST">
+            <input type="hidden" name="csrfmiddlewaretoken" :value="csrftoken" />
+            <button class="link" type="submit">{{cb.title}}</button>
+          </form>
+        </li>
+      </ul>
+    </template>
+  </Modal>
+</div>
+</template>
+
+<script>
+import Modal from "./Modal";
+
+function getCookie(name) {
+  var cookieValue = null;
+  if (document.cookie && document.cookie !== '') {
+    var cookies = document.cookie.split(';');
+    for (var i = 0; i < cookies.length; i++) {
+      var cookie = jQuery.trim(cookies[i]);
+      if (cookie.substring(0, name.length + 1) === (name + '=')) {
+        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+        break;
+      }
+    }
+  }
+  return cookieValue;
```

Comment:
I know this pattern is already in the code but there's:

```
export function get_csrf_token()
```

in `frontend/legacy/lib/helpers.js`. Do you think you could use that instead?

Reply:
Should I be importing the function like this to use it? The button doesn't seem to work when I do this though 🤔
<img width="784" alt="Screen Shot 2022-08-15 at 3 25 42 PM" src="https://user-images.githubusercontent.com/59305253/184703272-9f035133-04e0-4986-8144-0cfa71acf9cc.png">

Reply:
It doesn't take any arguments because it already knows what cookie name to look for. Does it work if you omit the argument? If not, don't worry about it and put it back the way you had it!

Reply:
Oh! Change the import to:

```
import { get_csrf_token } from "../../frontend/legacy/lib/helpers";
```

Reply:
great it works! I think this PR is ready now! :)